app/services/classification/base.rb

```diff
@@ -146,7 +146,7 @@ def path_elements(path = nil)
 
     def validate_lookup_value(key, value)
       lookup_value = key.lookup_values.build(:value => value)
-      return true if lookup_value.send(:validate_list) && lookup_value.send(:validate_regexp)
+      return true if lookup_value.send(:validate_value)
```

Comment:
It kind of has to be here, since we rely on late validation for ERB based values.  Could this use Parameters::Validator directly rather than calling a validation method?  It may require some changes to the interface to get rid of the model, or build a LookupValue still but then call Parameters::Validator directly with the newly constructed model instead of calling its validator.